pkg/ovs/openflow/ofctrl_packetout.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 // #nosec G404: random number generator not used for security purposes
-var pktRand = rand.New(rand.NewSource(time.Now().UnixNano()))
+var pktRand = rand.New(rand.NewPCG(uint64(time.Now().UnixNano()), 0))
```

Comment:
maybe we could use `rand.New(rand.NewPCG(rand.Uint64(), rand.Uint64()))` here. No need to use the `time` package anymore to seed?